gravitee-am-common/src/main/java/io/gravitee/am/common/audit/EventType.java

```diff
@@ -324,6 +324,12 @@ public interface EventType {
     String THEME_DELETED = "THEME_DELETED";
 
     String MFA_VERIFICATION_LIMIT_EXCEED = "MFA_VERIFY_LIMIT_EXCEED";
+    String MFA_CHALLENGE_SEND = "MFA_CHALLENGE_SEND";
+    String MFA_CHALLENGE_SUCCEED = "MFA_CHALLENGE_SUCCEED";
+    String MFA_CHALLENGE_FAILED = "MFA_CHALLENGE_FAILED";
+    String MFA_ENROLL_SUCCEED = "MFA_ENROLL_SUCCEED";
+    String MFA_ENROLL_FAILED = "MFA_ENROLL_FAILED";
```

Comment:
Event type should not include status. The status is already in the log.
It could be:
MFA_CHALLENGE_SEND,  MFA_CHALLENGE, MFA_ENROLL, MFA_MAX_ATTEMPT_REACHED